client/src/components/pages/ProfileUpdate.tsx

```diff
@@ -391,6 +407,7 @@ const Screen: FC = () => {
               text={getString('UPDATE')}
             />
           </StyledButtonWrapper>
+          <Snackbar testID="UpdateProfile" ref={snackbar} />
```

Comment:
For `dom` id, it is recommended to write in `kebab-case`.

Reply:
Also, how do you think about putting this in `RootStack` and use it in every component?

Reply:
I will change id of DOM follow your advice.
and I will attempt to put Component in RootStack